provisioning/common/etcd/values.yaml

```diff
@@ -65,21 +84,18 @@ networkPolicy:
           matchLabels:
             app: datadog-agent
 
+# Records are persisted
 persistence:
   enabled: true
   size: 512Mi
 
-tolerations:
-  - key: app
-    operator: Exists
-    effect: NoSchedule
-
+# At least 51% of the cluster must be available during rollout,
+# otherwise, the quorum is lost and the cluster will not work.
+# This should be disabled if replicaCount=1, that would block the rollout completely.
 pdb:
-  create: false
-
-nodeSelector:
-  nodepool: main
+  create: true
```

Comment:
Pre Buffer API musi mat etcd nastavene `pdb - 51%`.
Vzdy musi byt dostupne `>50%` clustra, aby zostalo zachovane `quorum`, tj. v nasom pripade su to `2 z 3`.

Bolo tu `create: false`, lebo pri `1 node`, je `51%` -> `1 node` a blokovalo to `rollout`.
Tu sa to neda vyriesit podmienkou, preto je podmienka v bash scripte.